.github/workflows/build.yml

```diff
@@ -44,7 +44,4 @@ jobs:
           java-version: 11
 
       - name: Run all device checks
-        uses: reactivecircus/android-emulator-runner@v2
-        with:
-          api-level: 29
-          script: ./gradlew connectedCheck --stacktrace
\ No newline at end of file
+        run: ./gradlew pixel2api29DebugAndroidTest --stacktrace
```

Comment:
Curious whether there is a direct alternative to `connectedCheck` here since it seems to include other verification tasks rather than just android tests

Reply:
I think this is what `pixel2api29Check` will do:

```
pixel2api29Check - Runs all device checks on the managed device pixel2api29.
```